src/test/java/com/parasoft/findings/utils/results/testableinput/LocationUtilTest.java

```diff
@@ -0,0 +1,90 @@
+package com.parasoft.findings.utils.results.testableinput;
+
+import com.parasoft.findings.utils.results.violations.LocationsException;
+import com.parasoft.findings.utils.results.xml.IXmlTagsAndAttributes;
+import org.junit.jupiter.api.Test;
+import java.util.Properties;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class LocationUtilTest {
+
+    @Test
+    public void testCreateTestableInput_projIdAndResProjPathAttributesInProperties() throws LocationsException {
+        Properties properties = new Properties();
+        properties.put("projId", "project");
+        properties.put("resProjPath", "test");
+
+        ITestableInput result = LocationUtil.createTestableInput(properties);
+
+        assertEquals("test", result.getName());
+        assertTrue(((FileTestableInput) result).getFileLocation().getAbsolutePath().contains("project\\test"));
+    }
+
+    @Test
+    public void testCreateTestableInput_scPathAttributeInProperties() throws LocationsException {
+        Properties properties = new Properties();
+        properties.put(IXmlTagsAndAttributes.SOURCE_CONTROL_PATH_ATTR, "project");
+        properties.put("test", "files");
+
+        ITestableInput result = LocationUtil.createTestableInput(properties);
+
+        assertEquals("project", result.getName());
+        assertEquals("files", ((FileTestableInput) result).getAttribute("test"));
```

Comment:
Are you trying to test the code block to add additional attributes?
I see similar test in `testCreateTestableInput_noSymIdAttributeInProperties` test method again.
I think it's better to remove the test from both places and create its own test method to make it clear.